ui/src/theme.tsx

```diff
@@ -0,0 +1,185 @@
+import { createTheme } from "@mui/material/styles";
+
+export const theme = createTheme({
```

Comment:
I think it will be an option for a javascript plugin that either wraps or replaces the argument to createTheme. Some high level things like logos and primary colors might make sense as part of the underlay or other config but full styling would require too much customization I think. I don't want to be recreating CSS in the underlay. Obviously anything related to the underlying data will still remain in the underlay though.